scripts/src/submission/submission.py

```diff
@@ -535,8 +533,11 @@ def is_valid_web_catalog_only(self, repo_path: str = "") -> tuple[bool, str]:
 
         return True, ""
 
+    def get_pr_number(self):
+        return self.api_url.split("/")[-1]
 
-def get_file_type(file_path):
+
+def get_file_type(file_path: str) -> tuple[str, re.Match[str]]:
```

Comment:
- The return value of this function has a typo ("Unknwown"). I can't select it for this comment because you didn't modify it here.